vertx-web/src/main/java/io/vertx/ext/web/Locale.java

```diff
@@ -23,46 +23,56 @@
  */
 
 import io.vertx.codegen.annotations.VertxGen;
-import io.vertx.ext.web.impl.LocaleImpl;
+import io.vertx.ext.web.impl.ParsableLanguageValue;
 
+/**
+ * @deprecated Use {@link LanguageHeader} instead
+ */
 @VertxGen
+@Deprecated
 public interface Locale {
```

Comment:
We try to be conservative with API changes, just wondering what if we kept this interface but extend `ParsedHeaderValue`?